R/aux_functions.R

```diff
@@ -29,16 +29,16 @@ inf_fn <- function(inc_samp = NULL, k = NULL) {
 #'
 #' @examples
 #' res <- scenario_sim(
-#'   n.sim = 10,
-#'   num.initial.cases = 1,
-#'   prop.asym = 0,
-#'   prop.ascertain = 0.2,
+#'   n_sim = 10,
+#'   num_initial_cases = 1,
```

Comment:
as these are changing anyway, I think `n` would be preferrable to `n_sim` - `n` is the typical variable number for "number of deviates" in the various r(unif|binom|...) functions.

then `initial_cases` would be better than `num_initial_cases` - no name collision